spec/models/action_plan_spec.rb

```diff
@@ -1,5 +1,53 @@
 require 'rails_helper'
 
 RSpec.describe ActionPlan, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context "statistics" do
```

Comment:
If you create a `ActionPlanStatisticsCalculator` service, this can be moved into a different file.